### ISSUE TITLE
Running the POC in standalone external Tomcat

### ISSUE THREAD

**Issue by vbalasub** [fromi/spring-google-openidconnect#3]
Good POC.  Thanks for putting this together, really appreciated.

I have 2 questions:
1) I am trying to run the POC in an external Tomcat and i always end up with 404. though the poc works fine when i run in embedded tomcat using Spring Boot.  
I followed this writeup 
https://spring.io/blog/2014/03/07/deploying-spring-boot-applications
to run the app in a standalone tomcat, i still have the issue.  Do you have any suggestions on this front?

2) I would also like to run your POC  in JDK1.7.  Is there a way to create the authenticationManager programatically, or i would have to use the XML.

Thanks for your help.

**Comment by fromi** [comment]
Hi, I'm happy to see that it is useful ;)

1) Hard to say with this information, I only deployed that on Heroku. Tomcat can deploy multiple applications, usually without additional information the war name becomes the root path for the application.

2) Spring Boot and Java Config is 100% compatible with Java 7. Just convert the lambda I used to create the authenticationManager that does nothing into a anonymous class or a inner class: just have a look at Spring 'OAuth2ClientAuthenticationProcessingFilter', it contains an inner class named 'NoopAuthenticationManager' which does exactly the same thing. Note that a lambda in Java 8 can always be converted into a anonymous class.

**Comment by vbalasub** [comment]
Thanks for the info.  I was able to implement this functinality with JDK 1.7 without Spring Boot.  I had to use web.xml with org.springframework.web.context.request.RequestContextListener in the listener. Perhaps there is a way to get rid of this infavor of code config, not done it yet..  
I have a question, you know of a way to clear out access&refresh tokens on the client side using Spring Security to force user to redirect to Google again?
